packages/components/src/Shield/Shield.tsx

```diff
@@ -45,7 +46,6 @@ Shield.propTypes = {
   impact: propTypes.oneOfType([propTypes.string, propTypes.number]),
   hasLabel: propTypes.bool,
   size: propTypes.string, // sm, md, lg and xl,
-  label: propTypes.bool,
```

Comment:
@Hyperkid123 I'm a bit confused about this small change as that specific `label` prop was not used anywhere and as it had a boolean type I thought it referred to `hasLabel` prop

Reply:
Yup, you are right. The component was not using the `label` prop :+1: